### DISCUSSION TITLE
[Question] Alternative for C++?

### DISCUSSION THREAD

**Discussion by Davilarek** [EliotVU/Unreal-Library#65] [Q&A]
Not really an issue, I just wanted to know if there is an alternative library for C++

**Comment by EliotVU** [comment]
Maybe [UPKUtils](https://github.com/wghost/UPKUtils), or UE Viewer (but not much of a library).

Alternatively you could try https://stackoverflow.com/questions/13293888/how-to-call-a-c-sharp-library-from-native-c-using-c-cli-and-ijw

**Reply by Davilarek** [reply]
`The tools are locked to work with certain versions of the engine only.` on UPKUtils
for calling c# methods from c++, that sounds awesome but sadly I need it cross-platform
